zebra-chain/src/transaction/arbitrary.rs

```diff
@@ -524,12 +524,13 @@ impl Arbitrary for Memo {
     type Strategy = BoxedStrategy<Self>;
 }
 
+/// Generates arbitrary [`LockTime`]s.
 impl Arbitrary for LockTime {
     type Parameters = ();
 
     fn arbitrary_with(_args: ()) -> Self::Strategy {
         prop_oneof![
-            (block::Height::MIN.0..=block::Height::MAX.0)
+            (block::Height::MIN.0..LockTime::MAX_HEIGHT.0)
```

Comment:
Typo: this skips `locktime::MAX_HEIGHT`, which should be a valid height lock.

```suggestion
            (block::Height::MIN.0..=LockTime::MAX_HEIGHT.0)
```